rules/Php81/Rector/ClassConst/FinalizePublicClassConstantRector.php

```diff
@@ -60,6 +60,16 @@ public function getNodeTypes(): array
      */
     public function refactor(Node $node): ?Node
     {
+        $parentClass = $this->betterNodeFinder->findParentByTypes($node, [Node\Stmt\Class_::class]);
```

Comment:
findParentType can be used:

```suggestion
        $parentClass = $this->betterNodeFinder->findParentType($node, Node\Stmt\Class_::class);
```